ansible/playbooks/tasks/aws-cluster-hana.yaml

```diff
@@ -22,10 +24,10 @@
   when: is_primary
   changed_when: false
 
-- name: Ensure maintainence mode is active
+- name: Ensure maintenance mode is active
   ansible.builtin.command:
-    cmd: crm configure property maintenance-mode=true
-  when: crm_maintainence_mode is false or crm_maintainence_mode == 'unknown'
+    cmd: crm maintenance on
```

Comment:
seems an important change not related to Angi. Isn't it?

Reply:
This is correctly set of the maintenance mode, not forcing it to the configuratin file.

Reply:
And the maintenance-mode  alue in configuration was changing along the verisions it's once 'on/off' then 'true/false' etc. This way is much safer.